### ISSUE TITLE
Customize sync frequency

### ISSUE THREAD

**Issue by sirio81** [owncloud/client#3185]
From this topic:
https://forum.owncloud.org/viewtopic.php?f=14&t=26195

Scenario: I'm sysncing a folder with large files that change oftest (e.g. photoshop file or large documents).
Problem: every time I save, the client sync the file, creating versions and using my bandwidth.

Possible solution: have the possibility to set how ofter the client checks for changes. (As default, it's ok the client behaves like now).

**Comment by guruz** [comment]
Which client version?

**Comment by sirio81** [comment]
1.8

**Comment by guruz** [comment]
This is actually not related to any sync frequency but is done automatically during file change.
Would a "pause" option in the menu help you?

CC @jancborchardt

**Comment by dragotin** [comment]
Yes, that reminds me that we wanted to discuss/add a global "Pause syncing" button instead of a individual one for each sync connection. In the light of multi account however, *global* probably means *by account*.

**Comment by jancborchardt** [comment]
Well, the real problem here is that we don’t do delta syncing yet. #179 (it’s not scheduled for any milestone btw)

Needing to press a »Pause syncing« button (or even worse, a setting for changing the sync frequency) would most likely result in people forgetting to enable it again or unsynced files.

Also, a »Pause syncing« button is a really bad solution here. While these big files are syncing you probably also work on other files – which you want to have synced. Now we would have to include some granularity in that pausing, and welcome to hell’s kitchen.

**Comment by dragotin** [comment]
@jancborchardt *no* in this case, sorry. Never forget https://dragotin.wordpress.com/2015/02/09/incremental-sync-in-owncloud/

**Comment by jancborchardt** [comment]
Ok sure. But still both solutions are a bit problematic, no?

Maybe if we make it a bit more automatic? Let’s say we detect such a big file which gets synced all the time on small changes, we notice that and only sync that file every second interval (or something similar)?

**Comment by sirio81** [comment]
I like jancborchardt suggestion.
It may be a flag "enable smart sync".
This option consider
- file size
- change frequency
The higher the file size, the more we skip syncning it.
E.g. 
file < 1M
  sync 1 time every 5 minutes (it doens't matter if in between it changes several times)
1M < file <5M
  sync 1 time every hour
file > 5M
 sync 1 time every 2 hours

This are values just to give the idea.

**Comment by guruz** [comment]
DropBox: 
![dropbox](https://cloud.githubusercontent.com/assets/959327/7492753/8cc142f8-f3fb-11e4-8e4e-81bb88a58890.jpg)

OneDrive:
![onedrive](https://cloud.githubusercontent.com/assets/959327/7492756/9488dc4e-f3fb-11e4-8035-6dbb13134be1.jpg)

**Comment by jancborchardt** [comment]
Huh, I totally forgot we _don’t_ have that thing in the menu yet. Of course yes, it should be there.

Btw, your Dropbox version seems outdated – the popover looks a lot different these days.
